front/src/components/Admin/Registration.vue

```diff
@@ -143,6 +175,18 @@ export default defineComponent({
         );
       },
     },
+    icalPasswordRequired: {
+      get(): boolean {
+        return this.icalPassword != '';
+      },
+      set(icalPasswordRequired: boolean) {
+        if (icalPasswordRequired) {
+          this.icalPassword = Math.random().toString(16).slice(2);
```

Comment:
Please fix this like https://github.com/DU4L/CTFNote/commit/b5e4a1894a030c9b735c7b1c2f1f4f1e78d0ea61